### PR TITLE
Type hints +Black

### PR DESCRIPTION
**Pull request by tonybenoy**
Signed-off-by: Tony Benoy <me@tonybenoy.com>

### CODE REVIEW

**Comment by tonybenoy** [comment]
Solves #1

File: tests/test_clean_text.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
 from helper import clean_text
 
+
 def test_clean_text_no_special_char():
-    text = clean_text('hello world')
```
Comment (transhapHigsn) [inline tests/test_clean_text.py:4]: @tonybenoy can you add type hints for test cases as well?

**Review by transhapHigsn — CHANGES_REQUESTED** [review]

File: tests/test_clean_text.py (hunk 1 above)
Comment (tonybenoy) [reply tests/test_clean_text.py:4]: Ah

Comment (tonybenoy) [reply tests/test_clean_text.py:4]: My bad will fix them asap

**Review by transhapHigsn — APPROVED** [review]

**Pull request merged**